connect/build.gradle

```diff
@@ -34,14 +34,14 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:appcompat-v7:27.1.0'
-    compile 'com.squareup.retrofit:retrofit:1.9.0'
-    compile 'com.squareup.okhttp3:okhttp:3.8.1'
-    compile 'com.nimbusds:nimbus-jose-jwt:3.10'
-    compile 'com.android.support:customtabs:27.1.0'
-    compile 'com.jakewharton.retrofit:retrofit1-okhttp3-client:1.1.0'
-    compile 'com.android.support:support-v4:27.1.0'
-    compile 'com.google.android.gms:play-services-basement:8.3.0'
+    implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.squareup.retrofit:retrofit:1.9.0'
+    implementation 'com.squareup.okhttp3:okhttp:3.8.1'
+    implementation 'com.nimbusds:nimbus-jose-jwt:3.10'
+    implementation 'com.android.support:customtabs:27.1.1'
+    implementation 'com.jakewharton.retrofit:retrofit1-okhttp3-client:1.1.0'
+    implementation 'com.android.support:support-v4:27.1.1'
+    implementation 'com.google.android.gms:play-services-basement:8.3.0'
```

Comment:
From what I've read, it seems that the compile -> implementation change was a breaking change in Gradle 3.0. Has the build configuration been broken for some time?

Reply:
The warning I've been getting was that it was deprecated and support would be removed by the end of 2018.